test/features/support/steps.js

```diff
@@ -0,0 +1,14 @@
+const { expect } = require('chai');
+const { Given, When, Then } = require("cucumber");
+
+const array = [1,2,3];
+let result = 0;
+
+Given("an Array with items", function() {
```

Comment:
you should define the array with items into this step

Reply:
Do they become work like the "describe"? I'm still not to sure how can i organize correctly the steps and tests

Reply:
something like this:
```js
let array;
Given("an Array with items", function() {
     array = [1,2,3];
});
```